src/dependencies.jl

```diff
@@ -461,8 +461,8 @@ function generate_steps(dep::LibraryDependency,h::Zypper,opts)
     end
     sudo = get(opts, :sudo, has_sudo[]) ? `sudo` : ``
     @build_steps begin
-        println("Installing dependency $(h.package) via `$(sudoname(sudo))zypper install $(h.package)`:")
-        `$sudo zypper install $(h.package)`
+        info("Installing dependency $(h.package) via `$(sudoname(sudo))zypper install -n $(h.package)`:")
```

Comment:
Oh, I read that it was `-n` for `--no-confirmation` or something like that. I can make it `-y`.